iodata/inputs/orca.py

```diff
@@ -0,0 +1,76 @@
+# IODATA is an input and output module for quantum chemistry.
+# Copyright (C) 2011-2019 The IODATA Development Team
+#
+# This file is part of IODATA.
+#
+# IODATA is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# IODATA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+# --
+"""Orca Input Module."""
+
+from typing import TextIO
+from string import Template
+from attr import asdict
+
+from ..iodata import IOData
+from ..periodic import num2sym
+from ..utils import angstrom
+
+__all__ = []
+
+
+default_template = """\
+! ${lot} ${obasis_name} ${run_type}
+# ${title}
+*xyz ${charge} ${spinmult}
+${geometry}
+*"""
+
+
+def write_input(f: TextIO, data: IOData, template: str = None):
+    """Do not edit this docstring. It will be overwritten."""
+    # load IOData dict using attr.asdict because the IOData class uses __slots__
+    fields = asdict(data, recurse=False)
+
+    # store atomic coordinates in angstrom
+    fields["atcoords"] = data.atcoords / angstrom
+
+    # replace None for default values
+    fields["title"] = data.title if data.title is not None else 'Input Generated by IOData'
+    fields["run_type"] = data.run_type if data.run_type is not None else 'energy'
+    # convert spin polarization to multiplicity
+    fields["spinmult"] = int(data.spinpol) + 1 if data.spinpol is not None else 1
+    fields["charge"] = int(data.charge) if data.nelec is not None else 0
```

Comment:
I use data.nelec instead of data.charge in this conditional, because I was getting the same error mentioned in #186 when nelec attribute is not assigned.
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/maarten/anaconda3/envs/iodata/lib/python3.8/site-packages/iodata/iodata.py", line 274, in charge
    return self.atcorenums.sum() - self.nelec
TypeError: unsupported operand type(s) for -: 'float' and 'NoneType
```
Was this fixed in that PR? If so, for change it to data.charge.

Reply:
Yes, it was fixed, but not merged yet. I'll make a separate PR with just the fix, wo we can merge it quickly.

Reply:
See #196.